### ISSUE TITLE
[Bug]: Can not unsecure site

### ISSUE THREAD

**Issue by isimmons** [beyondcode/herd-community#554]
### Platform

Windows

### Operating system version

Windows 10 Home 64 version 22H2

### System architecture

Windows

### Herd Version

1.01

### PHP Version

8.34

### Bug description

I clicked on the lock icon to secure my Laravel 11 example site. After clicking yes on 5 prompts to allow program to make changes, Herd secured the site successfully and shows the locked lock icon.

Then after some time I tried clicking the lock icon to unsecure the site. After clicking yes on 2 prompts to allow program to make changes, Herd appeared to be working on the task (red balls moving and slight dimming). Then it appeared to be finished but the site is still secured, serving over https and the lock icon is still locked.

I then tried using the cli (powershell as admin). The command appears to run successfully but the site is still secured.

![Screenshot (39)](https://github.com/beyondcode/herd-community/assets/1780886/f2c0fb2f-d7dd-46f4-8c16-ae66c872ec5e)

**Comment by mechelon** [comment]
Can you try to run the `herd unsecure` command inside the `Herd/example` folder? That would help us to narrow down the problem.

**Comment by isimmons** [comment]
Same result running inside herd/example. 

![Screenshot (40)](https://github.com/beyondcode/herd-community/assets/1780886/4fc6bf5f-81ca-4a48-ac98-248387ca9cfa)

**Comment by isimmons** [comment]
When I secure the site, the following files get created

![Screenshot (41)](https://github.com/beyondcode/herd-community/assets/1780886/19d1ceb7-e4ea-464f-ad26-14d13706b368)

If I manually delete the certificate files the site shows unsecured in Herd but loads with https in chrome. If I manually delete the NGINX conf file then the site can't be reached in chrome. I resecured the site so it would recreate the files and make everything work again. But none of these files exist on a new  unsecured site.

**Comment by bennajah** [comment]
same here

**Comment by quizafebrick** [comment]
Yeah, same here.

**Comment by kamaludin21** [comment]
same issue

**Comment by ktross** [comment]
Same issue here. I'm trying to unsecure it in hopes of getting `vite` to run.

I get this error when setting `detectTls: 'mysite.test'`
```
vite

error when starting dev server:
Error: Unable to find certificate files for your host [mysite.test] in the [C:\Users\kylet\.config\valet/Certificates] directory. Ensure you have secured the site via the Herd UI or run `valet secure`.
```

I'm on a fairly vanilla Windows 11 install. I just installed it fresh a couple weeks ago.

Edition	Windows 11 Pro
Version	23H2
Installed on	‎3/‎22/‎2024
OS build	22631.3296
Experience	Windows Feature Experience Pack 1000.22687.1000.0

**Comment by Kenneth-Inkum** [comment]
> Same issue here. I'm trying to unsecure it in hopes of getting `vite` to run.
> 
> I get this error when setting `detectTls: 'mysite.test'`
> 
> ```
> vite
> 
> error when starting dev server:
> Error: Unable to find certificate files for your host [mysite.test] in the [C:\Users\kylet\.config\valet/Certificates] directory. Ensure you have secured the site via the Herd UI or run `valet secure`.
> ```
> 
> I'm on a fairly vanilla Windows 11 install. I just installed it fresh a couple weeks ago.
> 
> Edition Windows 11 Pro Version 23H2 Installed on ‎3/‎22/‎2024 OS build 22631.3296 Experience Windows Feature Experience Pack 1000.22687.1000.0

Yes, I have the same issue as well. I had to manually copy the valet folder in the location where vite is looking for the certificate fir it to work.

**Comment by mpociot** [comment]
This is now fixed with Herd 1.1.0
